core/views/browse.py

```diff
@@ -53,8 +53,9 @@ class SelectYearForm(django_forms.Form):
     page_title = "Browse Issues: %s" % title.display_name
     page_name = "issues"
     crumbs = create_crumbs(title)
-    return render_to_response('issues.html', dictionary=locals(),
+    response = render_to_response('issues.html', dictionary=locals(),
                               context_instance=RequestContext(request))
+    return cache_tag(response, "lccn=%s" % lccn)
```

Comment:
This usage feels like it should be a decorator to match `cache_page`

Reply:
Also, the trailing indentation should match the parentheses following PEP-8

Reply:
How would the decorator get the LCCN?